--- ./opt/brother/Printers/DCPL2550DN/cupswrapper/lpdwrapper.orig	2017-07-18 03:34:35.000000000 +0200
+++ ./opt/brother/Printers/DCPL2550DN/cupswrapper/lpdwrapper	2018-05-06 13:56:44.395173882 +0200
@@ -72,16 +72,7 @@
 $ENV{NODENAME} = `uname -n`;
 
 
-my $basedir = Cwd::realpath ($0);
-if ( $basedir eq '' ){
-  $basedir = `readlink $0`;
-  if ( $basedir eq '' ){
-      $basedir = `realpath $0`;
-  }
-}
-chomp($basedir);
-
-$basedir =~ s/\/cupswrapper\/.*$//g;
+my $basedir = "/opt/brother/Printers/DCPL2550DN";
 
 
 
